release_notes/unreleased.md

```diff
@@ -1 +1,4 @@
 **Unreleased**
+* Removed 'pyc' files from the app tarball [PAPP-23233]
+* Updated the app documentationm [PAPP-10033]
+* Provide human readable error messages for errors [PAPP-10034]
```

Comment:
Can we update this point with below mentioned and please add FIPS related point
* Improved error message handling in the actions [PAPP-10034]
* Marked the app as FIPS Compliant

Reply:
Done

Reply:
not asking for adding fips_compliant label. I saw metadata in JSON so based on that please add a point in unreleased.md file. For example
<img width="933" alt="Screenshot 2021-12-31 at 12 38 05 PM" src="https://user-images.githubusercontent.com/64279254/147808999-cc31bb72-8419-4c4f-9304-b295ef24958f.png">
.

Reply:
Added note for FIPS Compliant app in unreleased.md file.